BuildMasterAutomation/Functions/Invoke-BMRestMethod.ps1

```diff
@@ -140,8 +140,14 @@ function Invoke-BMRestMethod
                 ForEach-Object { $_ } 
         }
     }
-    catch [Net.WebException]
+    catch
     {
+        $webExceptions = @( 'Microsoft.PowerShell.Commands.HttpResponseException', 'System.Net.WebException' )
+        if( $_.Exception.GetType().FullName -notin $webExceptions )
+        {
+            throw

```

Comment:
Should we throw the original exception message? E.g. `throw $_.Exception.Message` or maybe we can just `throw $_`?

Reply:
`throw` by itself throws the original exception, preserving the script stack trace and everything.

Reply:
I had no idea. Nifty!